src/pds_doi_service/core/actions/check.py

```diff
@@ -106,7 +106,7 @@ def _update_transaction_db(self, pending_record):
         lidvid = '::'.join([pending_record['lid'], pending_record['vid']])
```

Comment:
I realize this is out of the scope for the current review, but using `join` here feels a little out of the spirit of its intent: concatenating a sequence of perhaps unknown length with a separator. Since there's a constant (and small) number of items being concatenated (a `lid` and a `vid`), it might be clearer to just say:

```python
lidvid = pending_record['lid'] + '::' + pending_record['vid']`
```
or
```python
lidvid = f'{pending_record["lid"]::{pending_record["vid"]'
``` 
🤷‍♀️